switchboard/sql/SCHEMA.sql

```diff
@@ -133,8 +133,8 @@ create type job_known_state as enum (
     -- The job is currently running. In a precise sense, the real meaning is "the job MAY be running", however, the
     -- switchboard should treat "may be running" as "is running"; thus, this state is called 'running'.
     'running',
-    -- The job has run to completion.
-    'finished');
+    -- The job is not running and does not need to run.
+    'not_queued');
 create table jobs
```

Comment:
What would we need this state for exactly?

Reply:
The `known_state` type is intended to be used when the switchboard is reconstructing state after having to restart.